src/afl-forkserver.c

```diff
@@ -867,6 +868,11 @@ void afl_fsrv_start(afl_forkserver_t *fsrv, char **argv,
 
   }
 
+  if (getenv("AFL_PRELOAD_DISCRIMINATE_FORKSERVER_PARENT") != NULL)
```

Comment:
can you move this to afl_fsrv_init please and add the variable to the afl_forkserver_t struct?